third_party/dependency_analyzer/src/main/io/bazel/rulesscala/dependencyanalyzer/OptionsParser.scala

```diff
@@ -22,6 +22,42 @@ object OptionsParser {
 
     new OptionsParser(error = error, options = optionsMap)
   }
+
+  def decodeStringSeqOpt(targetsStr: String): Seq[String] = {
+    //Lists of items are demlimited by ';' allowing for escaped ';' (since ; is valid in a bazel label)
+   
+    def extractAndAppendToken(tokens:List[String], str:String, startIdx:Int, endIdx:Int) : List[String] = {
+      val token = str.substring(startIdx, endIdx).replace("\\", "")
+      if(!token.isEmpty()){
+        return tokens :+ token;
+      }
+      return tokens;
+    }
+
+    @annotation.tailrec
+    def tokenize(tokens:List[String], targetsStr:String, currTokenStartIdx:Int, currIdx:Int, isEscaped:Boolean) : List[String] = {
```

Comment:
nit: return type probably can be `Seq[String]`

Reply:
done